api/jobs.yaml

```diff
@@ -116,6 +131,45 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    required:
+      - required
+    properties:
+      required:
+        type: boolean
+        description: Whether or not authentication is required
+      client_id:
+        type: string
+        description: OAuth 2.0 client ID
+      scopes:
+        type: array
+        description: OAuth 2.0 requested scopes
+        items:
+          type: string
+  CapabilitiesResponse:
+    description: Specification of the backend's capabilities
+    required:
+      - authentication
+    properties:
+      authentication:
+        $ref: '#/definitions/AuthenticationCapability'
+      display_fields:
+        type: object
+        description: Fields on QueryJobsResult returned from POST /jobs/query
+          populated on some or all jobs. The fields are mapped to their display
+          names, in order of importance. The UI will attempt to display as
```

Comment:
For separation of concerns, I'd drop this UI sentence. It's up to the UI to decide what to do with this information.

Reply:
Done.